pkg/postgres/conn.go

```diff
@@ -81,8 +109,31 @@ func (c *Conn) QueryRow(ctx context.Context, sql string, args ...interface{}) *R
 func (c *Conn) SendBatch(ctx context.Context, b *pgx.Batch) *BatchResults {
 	ctx, cancel := contextutil.ContextWithTimeoutIfNotExists(ctx, defaultTimeout)
 
+	var batchResults pgx.BatchResults
+	if tx, ok := TxFromContext(ctx); ok {
+		batchResults = tx.SendBatch(ctx, b)
+	} else {
+		batchResults = c.PgxPoolConn.SendBatch(ctx, b)
+	}
+
 	return &BatchResults{
-		BatchResults: c.Conn.SendBatch(ctx, b),
+		BatchResults: batchResults,
 		cancel:       cancel,
 	}
 }
+
+// CopyFrom wraps pgxpool.Conn CopyFrom
+func (c *Conn) CopyFrom(ctx context.Context, tableName pgx.Identifier, columnNames []string, rowSrc pgx.CopyFromSource) (rows int64, err error) {
+	ctx, cancel := contextutil.ContextWithTimeoutIfNotExists(ctx, defaultTimeout)
+	defer cancel()
+
+	if tx, ok := TxFromContext(ctx); ok {
+		rows, err = tx.CopyFrom(ctx, tableName, columnNames, rowSrc)
+	} else {
+		rows, err = c.PgxPoolConn.CopyFrom(ctx, tableName, columnNames, rowSrc)
+	}
+	if err != nil {
+		incQueryErrors("copyfrom", err)
```

Comment:
do you think we should just make incQueryErrors not do anything if err == nil. That should make this a one liner?

Reply:
Yes, good point. I will make this small change.